Luke7787/ImmaculateInventors
# Feature/date

Summary:
-implemented dates to items when they are created

How to test:
-npm run dev
-add an item to the database
-check to see that it added the current date and time
